addons/website_livechat/tests/common.py

```diff
@@ -39,7 +42,13 @@ def setUp(self):
             'country_id': self.env.ref('base.de').id,
             'website_id': self.env.ref('website.default_website').id,
             'partner_id': self.env.ref('base.user_demo').partner_id.id,
-        }] + [visitor_vals]*self.max_sessions_per_operator)
+            'access_token': self.env.ref('base.user_demo').partner_id.id,
+        }] + [
+            dict(visitor_vals, **{
+                'access_token': hashlib.sha1(uuid.uuid4().hex.encode('utf-8')).hexdigest(),
+            })
```

Comment:
```suggestion
            dict(visitor_vals, access_token: uuid.uuid4().hex)
```
- a uuid4 is random data, hashing that is a complete waste of time.
- **-unpacking a literal dict is the same as just using a keyword parameter, except with more overhead